source/CommonJobs/CommonJobs.Domain/Person.cs

```diff
@@ -121,9 +121,9 @@ public string Id
         //Only in detailed view
         public string Skills { get; set; }
 
-        public virtual IEnumerable<SlotWithAttachment> AllAttachmentReferences
+        public virtual SlotWithAttachment[] AllAttachmentReferences
```

Comment:
Same comment as above. (I will mark them all so that later we can double check they're all resolved, one way or another.)